src/lib/trajectory/extract.ts

```
@@ -121,14 +122,26 @@ export const structural_data_extractor: TrajectoryDataExtractor = (
 
   if (frame.metadata) {
     // Extract other structural properties, avoiding volume duplicate
-    const structural_fields = [`density`, `temperature`]
+    const structural_fields = [`temperature`]
 
     for (const field of structural_fields) {
       if (
         field in frame.metadata &&
         typeof frame.metadata[field] === `number`
```

Biome (1.9.4)

[error] Invalid `typeof` comparison value: this expression is not a string literal

not a string literal

(lint/suspicious/useValidTypeof)